### PR TITLE
Initialize pro

### PR DESCRIPTION
**Pull request by KUBAHO3**
**In this project I have:**

- created a new project using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- Create a directory for your reusable components
- Install [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)
- [Setup my router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of App.js
- Create different components with their styles

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,229 +61,185 @@ After you're finished please remove all the comments and instructions!
...
 
+  - Figma is not available for now!
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](https://github.com/KUBAHO3/book-store/blob/events/MIT.md) licensed.
```
Comment (GzimAsani) [inline README.md]: - [x] MIT  link is leading to a broken page kindly fix it

File: src/components/Book.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  const percentage = 66;
+  return (
+    <div className="book-wrapper">
+      <div className="book-info">
+        <p className="book-genre">Action</p>
+        <p className="book-title">The Hunger games</p>
+        <p className="book-author">Sizanne collins</p>
+        <ul className="book-actions">
+          <li className="book-comment">Comments</li>
+          <li className="book-remove">Remove</li>
```
Comment (GzimAsani) [inline src/components/Book.js]: - [x] The remove needs to be a button, kindly update it

Comment (GzimAsani) [inline src/components/Book.js]: - [x] It is a requirement the book component to get the author and books as props and then render them

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @KUBAHO3             :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with the router:+1: 

 ## Required Changes 

- [x] Check the comments below :arrow_double_down:  

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers, and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (KUBAHO3) [reply README.md]: Thank you @GzimAsani I had added the working link

File: src/components/Book.js (hunk 2 above)
Comment (KUBAHO3) [reply src/components/Book.js]: Yeah sure done

Comment (KUBAHO3) [reply src/components/Book.js]: Now it is coming from parent component

**Review by amejid — APPROVED** [review]
Hi @KUBAHO3, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Gitflow :heavy_check_mark: 
- Props used to transfer data between components :heavy_check_mark: 
- Good file structure :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**